### DISCUSSION TITLE
issue with X-Server-API-Key with postal

### DISCUSSION THREAD

**Discussion by praction1988** [postalserver/postal#2920] [Help with using Postal]
I built up a postal mail server to send emails via API. According to the documentation, we should send the X-Server-API-Key along with the POST API request. I don't see any documentation on how to construct this. 

or under credinitial key value is the token to be used for API authentication.   When I used the key value as a token, I received an InvalidServerAPIKey error.

**Comment by willpower232** [comment]
In the web UI, you create Credentials within a server on this page

![image](https://github.com/postalserver/postal/assets/1619102/37770440-769a-4537-b3c6-68b2103d6e5a)

You will need to make sure Type is set to API when you create it.
